features/CLS000_Classification-association.feature

```diff
@@ -0,0 +1,15 @@
+@implementer-agreement
+@CLS
+@version1
+@E00020
+
+Feature: CLS000 - Classification Association
+    The rule verifies that there is an ability to classify elements, materials, and systems according to various classification systems,
+    such as the UNIFORMAT or Omniclass classification systems.
+
+    Scenario: Check for activation of Classification Association
+
+    Given an IfcClassificationReference
```

Comment:
```suggestion
    Given an IfcObjectDefinition
    Given its attribute HasAssociations
    Given its entity type is IfcRelAssociatesClassification
    Given its attribute RelatingClassification
    Given its entity type is IfcClassificationReference
```

I would write it like this mimicking the concept template and making sure that the ClassificationReference is meaningfully connected to elements in the model. Note that some of the entity type checks are probably redundant, because the attribute already strictly mandates that type.